### ISSUE TITLE
Variable Name Not Matching 

### ISSUE THREAD

**Issue by tomhuhh** [RuminantFarmSystems/RuFaS#813]
I noticed that the variable names in the variable_name.txt file do not match with the column names in the output cvs file. 

## Expected Behavior

I would expect the names are matching. 

## Current Behavior

For example, in the variable_name.txt, the variable name is 'LifeCycleManager.daily_update.life_cycle_daily_herd_update.values: num_cow_for_parity', but in the csv file, the corresponding column name is 'LifeCycleManager.daily_update.life_cycle_daily_herd_update.num_cow_for_parity' instead.

## Possible Solution

I was able to generate a list of column names from the output csv file using the code below 

`column_names = df.columns`

I was also able to filter the column names in the csv file by searching for a key word using the code below.  
```
# Define the keyword you want to search for
keyword = 'methane'

# Use a list comprehension to find column names containing the keyword
matching_columns = [col for col in df.columns if keyword in col]
```

## Steps to Reproduce

1.
2.
3.
4.

**Comment by ew3361zh** [comment]
@tomhuhh can you describe why it's important to you to have the names in the variable_name.txt file matching the column names in the csv output columns?

**Comment by tomhuhh** [comment]
@ew3361zh Thank you for your question. I certainly don't want to add more work to your plate, but I also aim to streamline the workflow, avoid redundancy, and simplify the process of evaluating RuFaS outputs, enabling us to receive quicker feedback during the development phase and stay on track to meet our milestones. Regarding the 'variable_names.txt' file, could you please remind me of its initial purpose? My challenge has been navigating through the extensive output CSV file, which contains over 1000 columns. The inconsistency in variable names (i.e., column names) makes it more challenging to filter the necessary variables. For instance, I mentioned the 'num_cow_for_parity' variable as an example. I believe that the situation could be improved with the implementation of graphs. However, my understanding is that graph implementation is currently of lower priority. In the meantime, we are expected to do all the post-processing analyses on our own. If I'm mistaken or overlooking any details, please feel free to clarify. Thank you for your hard work!

**Comment by ew3361zh** [comment]
>Regarding the 'variable_names.txt' file, could you please remind me of its initial purpose?

the variable_names.txt file idea originally stemmed from a request from @KFosterReed to be able to get a quick read on and report out on what data is being collected by OutputManager. I don't know if it's a regular part of Kristan's or anyone else's reporting (other than what you've described here). I tagged a bunch of the SMEs in Slack but not sure if that got buried among other things in the dev-team channel.

Do you know if anyone else currently uses the variable_names.txt file? And in what ways they use it?

>My challenge has been navigating through the extensive output CSV file, which contains over 1000 columns. The inconsistency in variable names (i.e., column names) makes it more challenging to filter the necessary variables. For instance, I mentioned the 'num_cow_for_parity' variable as an example. 

That makes sense. It's not a difficult adjustment if we do decide to make a change to have the csv columns and variable names in the file match.

>I believe that the situation could be improved with the implementation of graphs. However, my understanding is that graph implementation is currently of lower priority. In the meantime, we are expected to do all the post-processing analyses on our own. If I'm mistaken or overlooking any details, please feel free to clarify. Thank you for your hard work!

@allisterakun is working on this as we speak and is close to finishing. You can see the progress in #779.

**Comment by Isgotkowitz** [comment]
> Do you know if anyone else currently uses the variable_names.txt file? And in what ways they use it?

@ew3361zh I use the variables names file when I am debugging filter files. Sometimes I mess up the regex, and sometimes the variable I'm trying to get with the filter isn't even there, but having the variable names file makes the process waaay easier.

@tomhuhh is the behavior of the column names always ending in `.values` making things worse as well?

**Comment by tomhuhh** [comment]
> @tomhuhh is the behavior of the column names always ending in `.values` making things worse as well?

@Isgotkowitz Yep, the column names ending in .values did give me a hard time. 

@ew3361zh Thank you all for looking into this issue. After chatting with Kristan, we believe that instead of modifying the code to fit different use case scenarios, it would be helpful if Niko or anyone from the dev team could come up with a more clear instruction document to describe the common issues and the corresponding solutions when using the variable_names.txt file. Thanks again for your hard work!

**Comment by Isgotkowitz** [comment]
> Yep, the column names ending in .values did give me a hard time.

@tomhuhh this specific behavior could be adjusted pretty easily. The appending of `.values` is done to distinguish values of variables from info maps of variables, but when info maps are not being recorded the `.values` could be dropped without much danger of confusion. Would this behavior make things better for you, or do you usually generate CSVs with info maps?

**Comment by PooyaHekmati** [comment]
>>anyone from the dev team could come up with a more clear instruction document to describe the common issues 

@tomhuhh We first would need to know what these common issues are. I, at the least, still don't have a full picture of this issue.

>>is the behavior of the column names always ending in .values making things worse as well?

@Isgotkowitz if user wants to see info maps as well, we should make it possible.

@ew3361zh OM supports {"block", "inline", "verbose"} formatting options. Right now, we don't have a way to allow users to choose from this list. Can you add a GNU arg for it? If SME's prefer a fourth formatting option, we probably can expand this list.

**Comment by ew3361zh** [comment]
Addressed with PR #846.
